src/sage/schemes/projective/projective_morphism.py

```diff
@@ -688,31 +694,31 @@ def as_dynamical_system(self):
             sage: P.<x,y,z> = ProjectiveSpace(ZZ, 2)
             sage: H = End(P)
             sage: f = H([x^2, y^2, z^2])
-            sage: type(f.as_dynamical_system())
+            sage: type(f.as_dynamical_system())                                         # needs sage.schemes
             <class 'sage.dynamics.arithmetic_dynamics.projective_ds.DynamicalSystem_projective'>
 
         ::
 
             sage: P.<x,y> = ProjectiveSpace(QQ, 1)
             sage: H = End(P)
-            sage: f = H([x^2-y^2, y^2])
-            sage: type(f.as_dynamical_system())
+            sage: f = H([x^2 - y^2, y^2])
+            sage: type(f.as_dynamical_system())                                         # needs sage.schemes
             <class 'sage.dynamics.arithmetic_dynamics.projective_ds.DynamicalSystem_projective_field'>
 
         ::
 
-            sage: P.<x,y> = ProjectiveSpace(GF(5), 1)                                   # optional - sage.rings.finite_rings
-            sage: H = End(P)                                                            # optional - sage.rings.finite_rings
-            sage: f = H([x^2, y^2])                                                     # optional - sage.rings.finite_rings
-            sage: type(f.as_dynamical_system())                                         # optional - sage.rings.finite_rings
+            sage: P.<x,y> = ProjectiveSpace(GF(5), 1)
+            sage: H = End(P)
+            sage: f = H([x^2, y^2])
+            sage: type(f.as_dynamical_system())                                         # needs sage.rings.finite_rings sage.schemes
             <class 'sage.dynamics.arithmetic_dynamics.projective_ds.DynamicalSystem_projective_finite_field'>
```

Comment:
`GF(5)` is small.

Reply:
But perhaps dynamical systems needs extensions of `GF(5)`...

Reply:
I think this was just a leftover tag. Removed in 4eaca5d